### ISSUE TITLE
Support endless loop option for article order/TOC

### ISSUE THREAD

**Issue by fortes** [Treesaver/treesaver#209]
Feature request from mailing list:

> I would like to create an endless loop out of a series of articles, so that when you reach the last article, it brings you back to the
> first article.

**Comment by bramstein** [comment]
This is fairly easy to do with the new TOC/index code, the only "problem" is how we configure this feature, as I can imagine not everybody wants it turned on by default. I introduces a treesaver.config variable a while back, but I'm not sure that is the right thing to do (as it currently contains mostly obscure settings.)

**Comment by fortes** [comment]
Yes, we need an easy way to specify configuration options. I can think of a few:

### Config Variables

Something like:

```
<script>
  treesaver.config = { option: value };
</script>
```

May want to have helper setter/getter functions here in order not to clobber options. Note that we already have `TS_AUTO_METRICS` and a few others, would be good to clean up and use one method.

### Meta tags / HTML attributes

Requires no code and perhaps less error-prone, could read off of <meta> and other tags or attributes to pick up settings.

**Comment by fortes** [comment]
Created Issue #213 to track the config separately. As you mentioned though, this feature should be pretty easy (the looping, that is)
